master/buildbot/data/schedulers.py

```diff
@@ -55,6 +56,14 @@ def get(self, resultSpec, kwargs):
         defer.returnValue((yield self.db2data(dbdict))
                           if dbdict else None)
 
+    @defer.inlineCallbacks
+    def control(self, action, args, kwargs):
+        if action == 'setenabled':
```

Comment:
I am not very fond of the setenabled name.

I think this would be better to change 'setenabled' -> 'enable' and 'enabledValue' to 'enable'